protostar/commands/build_command.py

```diff
@@ -50,31 +65,26 @@ def arguments(self):
             BuildCommand.COMPILATION_OUTPUT_ARG,
         ]
 
-    async def run(self, args: Namespace):
-        await self.build(
-            output_dir=args.compiled_contracts_dir,
-            disable_hint_validation=args.disable_hint_validation,
-            relative_cairo_path=args.cairo_path,
-        )
+    async def run(self, args: Any):
+        write = self._messenger_factory.from_args(args)
```

Comment:
```suggestion
        messanger = self._messenger_factory.from_args(args)
```
It's strange `messanger_factory` produces `write`.

Reply:
Yeah, but it's also strange to have `messenger(...)` function (noun vs verb)

Reply:
`write_fn_factory`? I don't know. Let's leave it as it is.